[{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/144288274","html_url":"https://github.com/elastic/elasticsearch/issues/13855#issuecomment-144288274","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/13855","id":144288274,"node_id":"MDEyOklzc3VlQ29tbWVudDE0NDI4ODI3NA==","user":{"login":"xuzha","id":1799964,"node_id":"MDQ6VXNlcjE3OTk5NjQ=","avatar_url":"https://avatars0.githubusercontent.com/u/1799964?v=4","gravatar_id":"","url":"https://api.github.com/users/xuzha","html_url":"https://github.com/xuzha","followers_url":"https://api.github.com/users/xuzha/followers","following_url":"https://api.github.com/users/xuzha/following{/other_user}","gists_url":"https://api.github.com/users/xuzha/gists{/gist_id}","starred_url":"https://api.github.com/users/xuzha/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/xuzha/subscriptions","organizations_url":"https://api.github.com/users/xuzha/orgs","repos_url":"https://api.github.com/users/xuzha/repos","events_url":"https://api.github.com/users/xuzha/events{/privacy}","received_events_url":"https://api.github.com/users/xuzha/received_events","type":"User","site_admin":false},"created_at":"2015-09-30T05:49:44Z","updated_at":"2015-09-30T05:49:44Z","author_association":"CONTRIBUTOR","body":"@clintongormley , I just tested this against 2.0, seems changes in #11414 do catch this and throw `mapper_parsing_exception` [here](https://github.com/elastic/elasticsearch/blob/6885ab96805990d83e57dd0baded83252d7a2e0e/core/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java#L142): \n","performed_via_github_app":null},{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/comments/145035034","html_url":"https://github.com/elastic/elasticsearch/issues/13855#issuecomment-145035034","issue_url":"https://api.github.com/repos/elastic/elasticsearch/issues/13855","id":145035034,"node_id":"MDEyOklzc3VlQ29tbWVudDE0NTAzNTAzNA==","user":{"login":"clintongormley","id":56599,"node_id":"MDQ6VXNlcjU2NTk5","avatar_url":"https://avatars0.githubusercontent.com/u/56599?v=4","gravatar_id":"","url":"https://api.github.com/users/clintongormley","html_url":"https://github.com/clintongormley","followers_url":"https://api.github.com/users/clintongormley/followers","following_url":"https://api.github.com/users/clintongormley/following{/other_user}","gists_url":"https://api.github.com/users/clintongormley/gists{/gist_id}","starred_url":"https://api.github.com/users/clintongormley/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/clintongormley/subscriptions","organizations_url":"https://api.github.com/users/clintongormley/orgs","repos_url":"https://api.github.com/users/clintongormley/repos","events_url":"https://api.github.com/users/clintongormley/events{/privacy}","received_events_url":"https://api.github.com/users/clintongormley/received_events","type":"User","site_admin":false},"created_at":"2015-10-02T14:20:50Z","updated_at":"2015-10-02T14:20:50Z","author_association":"CONTRIBUTOR","body":"This is very curious.  #11414 was merged into beta1, but beta1 still fails to catch this malformed document.  However beta2 does throw an exception.  Not sure what else changed, but this appears to be fixed so I'll close\n","performed_via_github_app":null}]